packages/hierarchy-browser/src/CommunityCard/CommunityCard.tsx

```diff
@@ -26,35 +28,45 @@ export interface ICommunityCardProps {
 	level: number
 	hierachyDataProvider: HierarchyDataProvider
 	incrementLevel?: boolean // adjust from 0 to 1 based indexing on levels if needed
-	isOpen?: boolean
+	neighborsLoaded: boolean
+	neighborCallback?: ILoadNeighborCommunities
+	settings: ISettingState
 }
 
 const ENTITY_LOADER_MSG = 'Fetching entity data...'
 
 export const CommunityCard: React.FC<ICommunityCardProps> = memo(
 	function CommunityCard({
-		isOpen: isOpenProp,
 		community,
 		maxSize,
 		maxLevel,
 		level,
 		incrementLevel,
+		neighborsLoaded,
 		hierachyDataProvider,
+		neighborCallback,
+		settings,
 	}: ICommunityCardProps) {
-		const [dataProvider] = useState<CommunityDataProvider | undefined>(
+		const [dataProvider] = useState<CommunityDataProvider>(
```

Comment:
`useMemo<CommunityDataProvider>(() => new CommunityDataProvider(...), [/* no deps intentionally */])`

Reply:
It's a readability thing. useState implies that you have something that's going to be mutated by your logic. Because there's no mutation involved a useMemo communicates your intent better.